sentry-samples/sentry-samples-android/appcenter

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
can we call this into https://github.com/getsentry/sentry-java/blob/main/Makefile somehow?